test/plugins/GenerateKeras/GenerateKeras.spec.js

```diff
@@ -313,5 +314,17 @@ describe(pluginName, function () {
         });
     });
 
-    // TODO: test that we can run the given python code?
+    // Here, we test the actual generated code, by spawning a process in python
+    // The generated code should run without errors.
+    describe('run generated code', function () {
```

Comment:
Rather than naming the test suite what you are **doing**, name it based on what you are **testing**.

After looking over the test suite for this plugin, it might make more sense to add these as tests in the sections for each architecture ("multiple types of layer IO (seq2seq)", "nested (wrapped) layers", "multi arch output") as simply another call to `it` (no describe necessary).